internal/repos/store.go

```diff
@@ -217,12 +218,20 @@ func (s *store) trace(ctx context.Context, family string) (*trace.Trace, context
 
 func (s *store) CountNamespacedRepos(ctx context.Context, userID, orgID int32) (count uint64, err error) {
 	tr, ctx := s.trace(ctx, "Store.CountNamespacedRepos")
+	logger := trace.Logger(ctx, s.Logger)
```

Comment:
The following fields are (and should) be included on every message, so let's add them here and remove them fro the other log calls:

```suggestion
	logger := trace.Logger(ctx, s.Logger).With(log.Int("userID", int(userID)), log.Int("orgID", int(orgID)))
```